ibm_quantum_schemas/models/annotation_serializer.py

```diff
@@ -0,0 +1,120 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Serializer for samplomatic annotations."""
```

Comment:
This should probably move to samplomatic so that it has control over how its annotations are serialized universally (even though not needed for Executor).